Pa3/Durchführung/Project/app/State/InitializeMcu.c

```diff
@@ -36,11 +40,28 @@
 
 EventEnum InitializeMcu_InitializeAll(void)
 {
-
+  //HAL
+  if (Gpio_init(void) != 0)
```

Comment:
remember yoda, also use the enum values, they are in the header
```suggestion
  if (GPIO_RET_OK != Gpio_init())
```